src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -851,45 +865,25 @@ components:
         sentenceType:
           $ref: '#/components/schemas/SentenceType'
           description: The prisoner's sentence type.
```

Comment:
While the complete additionalNeeds summary may be useful in the future I think to make things simple I'll just add the columns that we know the UI needs. hence I've chopped out that additionalNeeds object.